src/ocean/sys/socket/AddressIPSocket.d

```diff
@@ -393,3 +405,76 @@ class AddressIPSocket ( bool IPv6 = false ) : IPSocket!(IPv6), IAddressIPSocketI
         this.ip_address_len = this.in_address.inet_ntop(this.ip_address_).length;
     }
 }
+
+/// Test that methods that accept IP addresses in presentation notation
```

Comment:
I believe using `///` here will make the following unittest be rendered as a usage example, which isn't what we want.